### ISSUE TITLE
publish a command line executable on releases

### ISSUE THREAD

**Issue by vpenades** [onnx/tensorflow-onnx#2321]
Before submitting your request, please review past submissions to ensure that it is not a duplicate of a known feature request.

### Describe the feature request

Right now, in order to use tf2onnx (and skl2onnx) you require python and all the tensorflow dependencies, which is quite bulky and cumbersome, _specially for those of us not used to work with python regularly_.

### Describe scenario use case

I use OnnxRuntime in C#, sometimes I only want to convert some model to ONNX, which is often, but not so regularly. Every time I have to do so I have to create the appropiate environment just to run what's supposed to be a command line tool.

So it could be desirable that, as long as the source code, to publish a packaged command line executable you just need to run with the appropiate arguments. I think that would greatly help all those that are not into python.

**Comment by vpenades** [comment]
> Didn't get your thoughts.
> 
> This tool supports a python command line to finish this work

Yes, but that command line command requires to have python runtime installed, and then create a python environment, download the required dependencies, etc, before using it.

And I understand these pre-requirements are a no-brainer to python developers, but for everybody else it's a painful. For example, the first time I tried to use this tool, I didn't know about python virtual environments, I installed the dependencies in the root, so I messed up the whole thing and I had to reinstall python from scratch.

So what I am asking is, as long as the regular python dependant releases, to release a friendly version of the command line tool, packaged as a _self contained, single executable_, that can be downloaded and run straight away.

**Comment by vpenades** [comment]
Seeing that this issue still has the "Pending on user response", let me know if I need to clarify my last comment

**Comment by fatcat-z** [comment]
No, no further information is needed. You've already clarified your requirement, thanks.
